### ISSUE TITLE
rename css classes

### ISSUE THREAD

**Issue by mccrmck** [mccrmck/mccrmck.github.io#12]
TL;DR - style items via composing w/ simpler (dvs. reusable) classes instead of making each class have complex and specific styling

consider the following (based on ideas from [this](https://adamwathan.me/css-utility-classes-and-separation-of-concerns/) article):

1. one .sass file for each page, using classes unique to elements on that page (dvs. `calendar-list-item`, `cv-list-item`
2. css classnames based on functionality and then an additional class for the unique elements (dvs. `summary-list-item calendar-item`)

I think I prefer the second option, but then I need a css file for each page (likely small) and then probably several (or one biggie) for the general classes...might also make things tricky to track down? Perhaps I can be smart about file naming/folder structure...
